### ISSUE TITLE
`conda info` cannot work with channels?

### ISSUE THREAD

**Issue by cswarth** [conda/conda#3268]
Is there any way to get `conda info` to provide information about packages from alternative channels?

I installed `raxml` from the `bioconda` channel,
```
$ conda install -c bioconda raxml

$ conda list raxml
# packages in environment at /home/cwarth/.conda/envs/py27:
#
Using Anaconda Cloud api site https://api.anaconda.org
raxml                     8.2.4                         1    bioconda

```

`conda info` cannot find any info about the `raxml` package,
```
$ conda info raxml
Fetching package metadata .......
Error: Package missing in current linux-64 channels: 
  - raxml
```

The `info` subcommand does not appear to have a `-c` option, so how does one get info about packages installed from alternative channels?

----
```
$ conda info
Current conda install:

             platform : linux-64
        conda version : 4.1.11
    conda-env version : 2.5.2
  conda-build version : 1.21.11+0.g5b44ab3.dirty
       python version : 2.7.12.final.0
     requests version : 2.10.0
     root environment : /home/matsengrp/local/anaconda2/2.4  (writable)
  default environment : /home/cwarth/.conda/envs/py27
     envs directories : /home/cwarth/.conda/envs
        package cache : /home/cwarth/.conda/envs/.pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/linux-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/linux-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /home/cwarth/.condarc
         offline mode : False
    is foreign system : False

```

**Comment by HugoTian** [comment]
Please try following

    conda config --add channels bioconda
    conda info raxml

**Comment by cswarth** [comment]
yup, that works, problem fixed.  Thank you!

Is that the way it should work though?  Could `conda info` can look at the installed package and find out what channel it came from?  

Or if `conda info` is supposed to be independent of what packages are installed, perhaps have it take a `-c channel` option like `conda install`?

**Comment by kalefranz** [comment]
Conda info definitely needs a -c option. On the todo list...

**Comment by mforbes** [comment]
Where is this "todo list"?  It still appears that this bug has not been resolved as of 4.4.8.

**Comment by kalefranz** [comment]
Use `conda search —info` in conda 4.4
